boskos/boskos.yaml

```diff
@@ -153,7 +153,7 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
       - name: boskos
-        image: gcr.io/k8s-prow/boskos/boskos:v20200213-a79910c4a
+        image: gcr.io/k8s-staging-boskos/boskos:v20200819-984516e
```

Comment:
There's no non-staging repo yet, so this is fine. (It's populated by images that are automatically built on merges, basically exactly the same as gcr.io/k8s-prow/boskos was.)

There will eventually hopefully be tagged releases in k8s.gcr.io/boskos, but we don't have those yet.